lessons/r/ggplot2/Lindsayggplot2_files/figure-markdown_github/Imagefiles.Rmd

```diff
@@ -0,0 +1 @@
+##Contains all image files
```

Comment:
These files will be renamed after including the above code chunk (`knitr::opt_chunk$set(fig.path = 'figure/')`)